src/test/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToLatexFormatterTest.java

```diff
@@ -45,7 +45,7 @@ public void testHTML() {
         assertEquals("{\\\"{a}}", formatter.format("&auml;"));
         assertEquals("{\\\"{a}}", formatter.format("&#228;"));
         assertEquals("{\\\"{a}}", formatter.format("&#xe4;"));
-        assertEquals("{$\\Epsilon$}", formatter.format("&Epsilon;"));
+        assertEquals("{{$\\Epsilon$}}", formatter.format("&Epsilon;"));
```

Comment:
OK. Talking about kerning problems one shouldn't use `{\"{a}}` etc at all
(which is why Biblatex converts to Unicode first). But again, choosing
between kerning problems and wrong symbol, I'd pick kerning problems any
day of the week.

And in that particular example the only potential kerning problems is for
Epsilon, not for ä, which is most likely not part of a word anyway (I have
a hard problem believing that Greek people write their references using
equations...).

It is indeed a bit annoying that the extra braces are only needed in the
title field though.